android/src/main/java/com/reactnativepiwikprosdk/PiwikProSdkModule.kt

```diff
@@ -83,6 +83,18 @@ class PiwikProSdkModule(reactContext: ReactApplicationContext) : ReactContextBas
         }
     }
 
+    @ReactMethod
+    fun setCustomDimension(index: Int, value: String, promise: Promise) {
+        try {
+            var track = TrackHelper.track()
+                .dimension(index, value)
+                .with(this.tracker ?: throw Exception("Tracker is not initialized"))
```

Comment:
Seems to cause compilation errors since dimensions cannot be set for trackers on Android without an event to attach them to.